codeHF/config_tasks.sh

```diff
@@ -314,6 +315,7 @@ function MakeScriptO2 {
   [ $DOO2_DPLUSDMINUS_DATA -eq 1 ] && WF_CORR="o2-analysis-hf-correlator-dplusdminus o2-analysis-hf-task-correlation-ddbar"
   [ $DOO2_DPLUSDMINUS_MCREC -eq 1 ] && WF_CORR="o2-analysis-hf-correlator-dplusdminus-mc-rec o2-analysis-hf-task-correlation-ddbar-mc-rec"
   [ $DOO2_DPLUSDMINUS_MCGEN -eq 1 ] && WF_CORR="o2-analysis-hf-correlator-dplusdminus-mc-gen o2-analysis-hf-task-correlation-ddbar-mc-gen"
+  [ $DOO2_D0 -eq 1 ] && WF_CORR="o2-analysis-hf-task-correlation-d0"
```

Comment:
Aren't `o2-analysis-hf-task-correlation-d0` and the tasks in `WF_CORR` independent?

Reply:
I thought it is like a category for all correlations. I changed it back.